tests/integration-tests-topologies/src/main/java/org/apache/pulsar/tests/topologies/PulsarCluster.java

```diff
@@ -271,13 +306,33 @@ public WorkerContainer getAnyWorker() {
     }
 
     public ExecResult runAdminCommandOnAnyBroker(String...commands) throws Exception {
+        return runCommandOnAnyBrokerWithScript(ADMIN_SCRIPT, commands);
+    }
+
+    public ExecResult runClientCommandOnAnyBroker(String...commands) throws Exception {
```

Comment:
who is using this method? if no one is using it, I'd prefer removing it for now.

Reply:
will remove it